extensions/amp-access/0.1/iframe-api/package.json

```diff
@@ -4,8 +4,21 @@
   "description": "IFrame API for amp-access",
   "author": "The AMP HTML Authors",
   "license": "Apache-2.0",
+  "main": "build/index.min.js",
+  "jsnext:main": "amp-iframe-api-export.js",
+  "scripts": {
+    "build": "./node_modules/.bin/rollup -c rollup.config.js && cp ./build/index.js ../../../../dist.3p/current/amp-iframe-api-export.js"
```

Comment:
But when you upload the package, how does this file actually ends up there? Also, it doesn't have to be in the `dist.3p`. If you prefer we can put it somewhere around here, just git-ignored.

Reply:
it will be there in `build` folder so I guess we can remove this